doc/source/bindings.rst

```diff
@@ -239,6 +239,7 @@ Returns values about kOS and hardware
     PRINT VERSION.            // Returns operating system version number. i.e. 0.8.6
     PRINT VERSION:MAJOR.      // Returns major version number. e.g. 0
     PRINT VERSION:MINOR.      // Returns minor version number. e.g. 8
+    PRINT VERSION:BUILD.      // Returns build version number. e.g. 2
```

Comment:
we have an example above `i.e. 0.8.6` then here we use `2` as the build number when we used `0` and `8` for major and minor. It would be nice to have them the same to make it clear that we are talking about part of the above concept.